hack/deployer/runner/kind.go

```diff
@@ -97,7 +96,7 @@ func (k *KindDriver) create() error {
 	if err != nil {
 		return err
 	}
-	defer os.Remove(tmpManifest.Name())
+	defer os.RemoveAll(tmpManifest.Name())
```

Comment:
Why this change? Isn't `tmpManifest` a file?

Reply:
I used a script to generate this PR, something may not have worked as expected here 😬  I'll fix it.